### PR TITLE
Add clarification for bug bounty folks

### PR DESCRIPTION
**Pull request by wslack**
So much spam of people trying to do SQL injection via the form. :/

### CODE REVIEW

File: pages/index.md [hunk 1]
```diff
@@ -41,6 +41,6 @@ Here are a few examples of Federalist websites in production:
 ![18f.gsa.gov]({{site.baseurl}}/uploads/partner-sites/18f-gsa-gov.png)
 **[18F Homepage](https://18f.gsa.gov/)**
 
-## Getting started
+## Purchasing Federalist access for your office
 
-If you have a project that you think would be a good fit for Federalist, [please let us know](https://docs.google.com/forms/d/1iB8aW7c9r1QH3s8XElQCrnXRGjAiPUYpWG1CMeEqGIo/viewform).
+If you have a project that you think would be a good fit for Federalist, [please let us know](https://docs.google.com/forms/d/1iB8aW7c9r1QH3s8XElQCrnXRGjAiPUYpWG1CMeEqGIo/viewform). (not for bug bounty participants)
```
Comment (jseppi) [inline pages/index.md]: Maybe just move the `.` after the closing parenthesis.

File: pages/index.md [hunk 2]
```diff
@@ -33,14 +31,8 @@ Possible sites include:
...
 - A single page with information about a recently released report that may receive a lot of traffic.
 
-Here are a few examples of Federalist websites in production:
-
-![College Scorecard]({{site.baseurl}}/uploads/college-scorecard.png)
-**[The College Scorecard](https://collegescorecard.ed.gov/)**
-
-![18f.gsa.gov]({{site.baseurl}}/uploads/partner-sites/18f-gsa-gov.png)
-**[18F Homepage](https://18f.gsa.gov/)**
+For real world examples, see these [ten example sites]({{site.baseurl}}/pages/about-federalist/example-sites/) of the 79 sites currently live on Federaist.
```
Comment (jseppi) [inline pages/index.md]: typo: `Federaist` -> `Federalist`

Comment (wslack) [reply pages/index.md]: is there an easier way to edit this than going to the codebase, changing my branch, and then manually clicking in to edit? That workflow is so annoying.

Comment (jseppi) [reply pages/index.md]: yeah, in the PR's "Files changed" tab you can click on the pencil icon to open the editor again:

screen shot 2017-08-29 at 9 54 07 am

**Pull request merged**